app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import (VaccineError,
+                        NotWearingMaskError)
```

Comment:
According to the checklist, when importing multiple classes/modules, it is preferable to use parentheses. Here is how you can improve your import statement: 

python
from app.errors import (
    VaccineError,
    NotWearingMaskError
)
```

Reply:
agree

Reply:
but not a valid comment as you already use parentheses